srp_test.go

```diff
@@ -418,6 +418,38 @@ func TestCalculateU(t *testing.T) {
 	}
 }
 
+func TestSRP_MarshalBinary(t *testing.T) {
+	v := big.NewInt(42)   // value doesn't matter for this test
+	k := big.NewInt(2038) // End of the world. Value doesn't matter
```

Comment:
Ideally I'd like to test the marshal and unmarshal with most of the values populated. Any suggestion on a good way to do this? Otherwise I'll likely just add something for the first few exchanges between the client and server.

Reply:
I redid the test to cover the flow in the example file. This actually presented an issue where `group` was not being marshalled properly. So I've also implemented the binary marshal and unmarshal interface on the Group type as well.